ui-v2/tests/acceptance/dc/acls/roles/sorting.feature

```diff
@@ -0,0 +1,58 @@
+@setupApplicationTest
+@notNamespaceable

```

Comment:
Just incase you don't spot, we removed this in the other PR, so probably do the same here.